.github/renovate.json5

```diff
@@ -41,10 +60,11 @@
   timezone: "Etc/UTC",
   automergeSchedule: "after 1 am and before 7 am",
   schedule: "every weekend",
-  prConcurrentLimit: 5, // No more than 5 open PRs at a time.
+  //prConcurrentLimit: 5, // No more than 5 open PRs at a time.
+  prConcurrentLimit: 0, // <-- TEMPORARY!
```

Comment:
what does this do?

Reply:
Unlimited concurrent PRs. It appears to be recommended for when you start if there's a lot of stuff out of date and you have the CI capacity.